README.md

```diff
@@ -74,6 +74,17 @@ Instead you can use `destruct` to destructure the data you want. This does the s
 {:data1, :data2} |> destruct({d1, d2} ~> d1) |> some_func(d2)
 ```
 
+To simply save a partial result for later use, consider using `leak/2`:
+
+```elixir
+[:data1, :data2] |> Enum.reverse |> leak(reversed) |> hd
+hd # => :data2
```

Comment:
make this look more like an iex terminal to make it more understandable... something like:

```elixir
iex> [:data1, :data2] |> Enum.reverse |> leak(reversed) |> hd
:data2
iex> reversed
[:data2, :data1]
```